src/index.ts

```diff
@@ -23,7 +23,16 @@ export const init = (options?: TtvcOptions) => {
 
   calculator = getVisuallyCompleteCalculator();
   void calculator.start();
-  window.addEventListener('locationchange', () => void calculator.start(performance.now()));
+
+  // restart measurement for SPA navigation
+  window.addEventListener('locationchange', (event) => void calculator.start(event.timeStamp));
```

Comment:
Is there more documentation on the `locationchange` event? I can't find it on MDN or in our repo